content/pages/burials-and-memorials/burial-planning.md

```diff
@@ -27,6 +27,7 @@ Yes. You can plan ahead to make the process of applying for a burial in a VA nat
 - Send it by U.S. mail to: 
   
 <dl class="va-address-block">
+    <dt></dt>
```

Comment:
Yes, I thought that was weird too. There are ~15 `va-address-block` elements in vets-website, so I'd rather file at ticket against the content team to update those and not do it here if that's OK.